README.md

```diff
@@ -1,2 +1,70 @@
-## Book Store CMS
```

Comment:
- [ ] It will be nice to add a project title and a description so someone visiting your repo for the first time may know what the project is about. 
- [ optional ] You can use the ReadMe [template](https://github.com/microverseinc/readme-template) provided by Microcerse to quickly setup